tests/Predis/Connection/Cluster/RedisClusterTest.php

```diff
@@ -1212,11 +1213,12 @@ public function testFetchSlotsMapFromClusterWithClusterSlotsCommand(): void
 
     /**
      * @group disconnected
+     * @dataProvider onMovedResponsesDataProvider
      */
-    public function testAskSlotMapToRedisClusterOnMovedResponseByDefault(): void
+    public function testAskSlotMapToRedisClusterOnMovedResponseByDefault(string $movedErrorMessage): void
     {
         $cmdGET = Command\RawCommand::create('GET', 'node:1001');
-        $rspMOVED = new Response\Error('MOVED 1970 127.0.0.1:6380');
+        $rspMOVED = new Response\Error($movedErrorMessage);
```

Comment:
@vladvildanov I just noticed that we're hardly performing ANY cluster operations. Should we see if we can run some of the command tests against cluster?

I gotta say, I hate cluster and never use it.

Reply:
That's an interesting statement, how do you solve fail-over in your use-cases without cluster? Or is it that you just don't need a fail-over in your use-cases?

Reply:
We just don't use any HA Redis setups.